packages/shared/src/graphql/njord.ts

```diff
@@ -213,3 +217,43 @@ export const getTransactions = async ({
 
   return result.transactions;
 };
+
+export const PRODUCTS_SUMMARY_QUERY = gql`
+  query productSummary($userId: ID!, $limit: Int = 24, $type: ProductType!) {
+    productSummary(userId: $userId, limit: $limit, type: $type) {
```

Comment:
`productSummary` sounds a bit strange when it takes userId as an argument, no?

Reply:
was thinking of naming it userProductSummary, think thats better?

Reply:
renamed